LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -732,79 +737,56 @@ - (void)updateViewConstraints {
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _animatingView
-                                     attribute: NSLayoutAttributeCenterX
-                                    multiplier: 1.0f
-                                      constant: _horizontalGap - 2.0f];
-        NSLayoutConstraint *firstDigitY =
-        [NSLayoutConstraint constraintWithItem: _firstDigitTextField
-                                     attribute: NSLayoutAttributeCenterY
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _enterPasscodeLabel
-                                     attribute: NSLayoutAttributeBottom
-                                    multiplier: 1.0f
-                                      constant: _verticalGap];
-        NSLayoutConstraint *secondDigitY =
-        [NSLayoutConstraint constraintWithItem: _secondDigitTextField
-                                     attribute: NSLayoutAttributeCenterY
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _enterPasscodeLabel
-                                     attribute: NSLayoutAttributeBottom
-                                    multiplier: 1.0f
-                                      constant: _verticalGap];
-        NSLayoutConstraint *thirdDigitY =
-        [NSLayoutConstraint constraintWithItem: _thirdDigitTextField
-                                     attribute: NSLayoutAttributeCenterY
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _enterPasscodeLabel
-                                     attribute: NSLayoutAttributeBottom
-                                    multiplier: 1.0f
-                                      constant: _verticalGap];
-        NSLayoutConstraint *fourthDigitY =
-        [NSLayoutConstraint constraintWithItem: _fourthDigitTextField
-                                     attribute: NSLayoutAttributeCenterY
-                                     relatedBy: NSLayoutRelationEqual
-                                        toItem: _enterPasscodeLabel
-                                     attribute: NSLayoutAttributeBottom
-                                    multiplier: 1.0f
-                                      constant: _verticalGap];
-        [self.view addConstraint:firstDigitX];
-        [self.view addConstraint:secondDigitX];
-        [self.view addConstraint:thirdDigitX];
-        [self.view addConstraint:fourthDigitX];
-        [self.view addConstraint:firstDigitY];
-        [self.view addConstraint:secondDigitY];
-        [self.view addConstraint:thirdDigitY];
-        [self.view addConstraint:fourthDigitY];
+        
+        float delta = 10.0f;
+        
+        for (int i = 0; i < _digitsCount; ++i) {
+            
+            float constant = _digitsCount % 2 == 0
```

Comment:
About constraints between fields. If I add constraint between `fieldX` and `fieldX-1` I will need to change calculation of constants (they are calculated based on the centre of the screen). So I prefere to keep constraint between `fieldX` and `_animatingView `. But If it's needed I can change calculation.